app/src/main/java/org/simple/clinic/contactpatient/ContactPatientUpdate.kt

```diff
@@ -31,9 +31,16 @@ class ContactPatientUpdate(
       is NextReminderDateClicked -> selectNextReminderDate(model)
       is PreviousReminderDateClicked -> selectPreviousReminderDate(model)
       is ManualDateSelected -> updateWithManuallySelectedDate(event, model)
+      is AppointmentDateClicked -> showManualDatePicker(model)
     }
   }
 
+  private fun showManualDatePicker(model: ContactPatientModel): Next<ContactPatientModel, ContactPatientEffect> {
+    val datePickerBounds = model.potentialAppointments.first().scheduledFor..model.potentialAppointments.last().scheduledFor
```

Comment:
should we separate the first and last `scheduledFor` to make it more readable? At first, it wasn't apparent it's a range between two dates. WDYT?

Reply:
Makes sense, I'll do this.